### ISSUE TITLE
Create Scribe-Data Pidgin data process queries

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#220]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would create the queries for [Pidgin languages](https://en.wikipedia.org/wiki/Pidgin) in the [src/scribe_data/language_data_extraction/Pidgin](https://github.com/scribe-org/Scribe-Data/tree/main/src/scribe_data/language_data_extraction/Pidgin) directory. To start we can make a nouns query and a verbs query in two separate PRs, and from there we can make new issues for other types of data. These queries can be based on the already existing queries for other languages 😊

### Contribution

Happy to support with this! Please let me know if there are questions, and I can also review once a PR is ready :)

**Comment by DeleMike** [comment]
Hi @andrewtavis I would love to work on this, please.

**Comment by DeleMike** [comment]
But what kind of pidgin? There are [different types of pidgin](https://en.wikipedia.org/wiki/Pidgin#List_of_notable_pidgins)
I am familiar with [Nigeria Pidgin](https://en.wikipedia.org/wiki/Nigerian_Pidgin)

**Comment by andrewtavis** [comment]
Hmmmmm, great point, @DeleMike :) You can work on Nigeria Pidgin, but then should we name the directory this as well?

**Comment by DeleMike** [comment]
Yes, great point too @andrewtavis!  To avoid ambiguity, we should do that.

We could have:
```bash
language_data_extraction/
    Nigerian_Pidgin/
    Other_Types_Of_Pidgin/
```

what do you think? If you agree, do we change the issue name or create a new one?

**Comment by andrewtavis** [comment]
Let's first also check to see which versions on Pidgin have data on Wikidata with [src/scribe_data/check_language_data.sparql](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/check_language_data.sparql). I'd say that the best naming criteria would also maybe be `Pidgin_Nigerian` so that in the directory structure all of the Pidgin versions will be together. We can also just create individual directories for individual types of Pdigin as is needed :)

**Comment by DeleMike** [comment]
Yes, you are right! We should use that convention and thanks for the link!

I just checked it out and compared the `Pidgin_Nigerian` to  `English`; it does not have a lot of data. is there a way we can improve this?

<img width="1383" alt="English Data Result Screenshot" src="https://github.com/user-attachments/assets/f6c7b611-8b48-4951-856d-7dc686ab4d69">
<img width="1389" alt="Nigerian Pidgin Data Result Screenshot" src="https://github.com/user-attachments/assets/09a4a033-72b3-4d05-982c-f5ad51c78c9b">

**Comment by andrewtavis** [comment]
Certainly the data can be improved via adding lexemes (language data) to Wikidata. Do you want to check other versions of Pidgin to see if there's another one that we could work on for now, and we could maybe do a verbs query for Nigerian Pidgin?

**Comment by DeleMike** [comment]
I will definitely check out other versions of Pidgin, but for now, can I work on the verbs query for Nigerian Pidgin?

**Comment by andrewtavis** [comment]
Yes, this would be fine @DeleMike :)

**Comment by DeleMike** [comment]
Nice! do I create an issue for this and work on a PR resolving the issue?

**Comment by andrewtavis** [comment]
No need for a new issue :) You can just work from this one and reference it in your PR.

**Comment by DeleMike** [comment]
Alright, thanks!

**Comment by andrewtavis** [comment]
Updated the description given the sub languages idea, @DeleMike :) Let me know what you think on this!

**Comment by DeleMike** [comment]
@andrewtavis, yes, that's a great update!
do we go ahead with creating a PR for noun queries? for the Nigerian Pidgin, I mean.

**Comment by DeleMike** [comment]
Also, I don't know if this creates a new issue, but, how do we make sure that the data returned from the extraction is tangible enough? For example, English returns probably over 1000+ words for noun queries. How can we ensure the data extracted from a language is tangible and sufficient to be useful?

**Comment by andrewtavis** [comment]
I think on our end we don't need to make sure it's tangible. It's enough to show people what's available so that people can then see that data needs to be added 😊

**Comment by andrewtavis** [comment]
And yes we can start on nouns!

**Comment by DeleMike** [comment]
Nice. will work on that!
